Sources/Algorithm.swift

```diff
@@ -669,11 +678,15 @@ internal struct TableKey<T: Hashable>: Hashable {
         return lhs.hashValue == rhs.hashValue
             && (lhs.pointer.distance(to: rhs.pointer) == 0 || lhs.pointer.pointee == rhs.pointer.pointee)
     }
+    
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(pointer.pointee)
```

Comment:
Looks good.
However, I'm concerned about the performance impact of this.
Can you verify it?

Reply:
@ra1028 https://github.com/apple/swift-evolution/blob/master/proposals/0206-hashable-enhancements.md It is somehow supposed to have better performance.

Reply:
Yes there should be no negative effect here.